### PR TITLE
Add column search feature

### PR DESCRIPTION
**Pull request by aireater**
Sometimes I know what the column name but I'm not sure what table the column is in. 

The database I work with is fairly large and I'm new to it. 

Maybe someone else will find this feature useful.

### CODE REVIEW

**Comment by aireater** [comment]
I went with .search col COLMN for the column command.

File: README.md [hunk 1]
```diff
@@ -59,20 +59,19 @@ mssql>
...
-.indexes TABLE  Lists all the indexes of a table
-.analyze        Analyzes the database for missing indexes.
-.quit           Exit the cli
+command                               description                                                     
+------------------------------------  ----------------------------------------------------------------
+.help                                 Shows this message                                              
+.databases                            Lists all the databases                                         
+.tables                               Lists all the tables                                            
+.sprocs                               Lists all the stored procedures                                 
+.search text TEXT or col COLUMN_NAME  Search text in all fields or search col to find column in tables
```
Comment (hasankhan) [inline README.md]: .search TYPE VALUE

Searches for a value of specific type (col|text)

Comment (hasankhan) [inline README.md]: We can also update readme to give an example of how to use search command, just like we have examples for some other commands

**Comment by hasankhan** [comment]
Thank you

**Pull request merged**